back/src/main/kotlin/com/sos/smartopenspace/domain/Track.kt

```diff
@@ -0,0 +1,25 @@
+package com.sos.smartopenspace.domain
+
+import org.hibernate.validator.constraints.Length
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+import javax.persistence.Id
+import javax.validation.constraints.NotEmpty
+import javax.validation.constraints.Size
+
+@Entity
+class Track(
+    @field:NotEmpty
+    val name: String,
+
+    @field:Size(max=500)
+    val description: String = "",
+
+    @field:Size(min=7, max = 7)
+    val color: String,
```

Comment:
por ahí se podría agregar un método `validate()` custom que haga un chequeo más estricto, onda respetar el `#A0B4E3` como color (onda [ésto](https://stackoverflow.com/questions/1636350/how-to-identify-a-given-string-is-hex-color-format)